test/utils/redux.js

```diff
@@ -12,10 +13,13 @@ function restoreAppStore() {
 }
 
 export function createMockedStore(sinon, mockedState = {}) {
-    const store = createStore(() => mockedState);
+    const middlewares = [thunkMiddleware];
+    const mockStore = configureStore(middlewares);
+    const store = mockStore(mockedState);
 
     sinon.spy(store, 'dispatch');
     store.restore = restoreAppStore;
+    store.subscribe = () => Function();
```

Comment:
@liyefei737 This parts of the code creates a mocked store, meaning that it will fake the actual store and provide an implementation that's easier to test.

In this case, I changed the `subscribe` function to make it return an empty function because `redux-mock-store` was making it return `null`.

Maybe the code is confusing (perhaps I should change that part)

```js
store.subscribe = () => Function();
// is equivalent to...
store.subscribe = function() {
    return function() {};
};
```